README.md

```diff
@@ -8,22 +8,24 @@
 
 [![status](<https://jenkins-samba.apps.ocp.cloud.ci.centos.org/buildStatus/icon?job=samba_build-rpms-centos9-master&subject=master / CentOS 9>)](https://jenkins-samba.apps.ocp.cloud.ci.centos.org/job/samba_build-rpms-centos9-master/) [![status](<https://jenkins-samba.apps.ocp.cloud.ci.centos.org/buildStatus/icon?job=samba_build-rpms-centos9-v4-19-test&subject=v4-19-test / CentOS 9>)](https://jenkins-samba.apps.ocp.cloud.ci.centos.org/job/samba_build-rpms-centos9-v4-19-test/) [![status](<https://jenkins-samba.apps.ocp.cloud.ci.centos.org/buildStatus/icon?job=samba_build-rpms-centos9-v4-20-test&subject=v4-20-test / CentOS 9>)](https://jenkins-samba.apps.ocp.cloud.ci.centos.org/job/samba_build-rpms-centos9-v4-20-test/)
 
-This repository contains automation to create RPMs for CentOS Stream 8/9 and
-Fedora from Samba repository. In order to allow building from a variety of git
+This repository contains automation to create [Samba](https://www.samba.org/) RPMs for CentOS Stream 8/9 and
+Fedora from the upstream  [code repository](https://git.samba.org/samba.git). In order to allow building from a variety of git
 refspecs, the following make target format is used:
 
 `$ make < rpms.centos8 | rpms.centos9 | rpms.fedora > [ vers=< fedora-version > refspec=< branch-name | tag-name | h:<git-commit-hash> > ]`
 
-Few examples:
+A Few examples:
 
-- `$ make rpms.centos8 refspec=v4-20-test`
-- `$ make rpms.centos9 refspec=h:a0862d6d6de`
-- `$ make rpms.fedora vers=39 refspec=samba-4.19.6`
+```console
+$ make rpms.centos8 refspec=v4-20-test
+$ make rpms.centos9 refspec=h:a0862d6d6de
+$ make rpms.fedora vers=39 refspec=samba-4.19.6
+```
 
-As of now 4.19, 4.20 and master branches are supported. In the absence of
+As of now, versions  4.19 and  4.20 and the master branch are supported. In the absence of the 
 *refspec* argument, the master branch is built by default. The above format is
 also applicable for other `make` targets. The *vers* argument is only valid for
-Fedora related make targets and in its absence, the default version is
+Fedora related make targets and specifies the Fedora version to build for. In its absence, the default version is
```

Comment:
(trying that again)  No comma between "absence" and "the default"

Reply:
@phlogistonjohn  wrote: 

> (trying that again) No comma between "absence" and "the default"

removed.